hypergan/loaders/image_loader.py

```diff
@@ -14,16 +14,21 @@ def build_labels(dirs):
     labels[dir.split('/')[-1]]=next_id
     next_id+=1
   return labels,next_id
-def labelled_image_tensors_from_directory(directory, batch_size, channels=3, format='jpg', width=64, height=64, crop=False, preprocess=False):
-  filenames = glob.glob(directory+"/**/*."+format)
-  labels,total_labels = build_labels(sorted(filter(os.path.isdir, glob.glob(directory+"/*"))))
+def labelled_image_tensors_from_directory(directory, batch_size, channels=3, format='jpg', width=64, height=64, crop=False, preprocess=False, filterX=None):
```

Comment:
Not part of this PR.  The --align flag was added, but it can be ignored for now.